examples/sdk-app-example/build.gradle

```diff
@@ -47,39 +35,25 @@ android {
             signingConfig signingConfigs.config
         }
     }
-    lintOptions {
-        disable "OnClick"
-        abortOnError true
-    }
-    externalNativeBuild {
-        cmake {
-            path "CMakeLists.txt"
-        }
-    }
+    lintOptions.disable "OnClick"
+    externalNativeBuild.cmake.path "CMakeLists.txt"
 }
 
 // set this value to true if you wish to test with a local artefact
 // it will instruct gradle to copy the SO files from the AAR to the correct build directory
 def usingLocalArtefact = false
 
 dependencies {
-    if (usingLocalArtefact) {
```

Comment:
This was kind of handy (and `usingLocalArtefact` is still defined above)

Reply:
This has caused quite a bit of confusion in other people who have used the notifier's example app. This method also occasionally fails to find the SO files and fails to build the entire project, which hasn't been reproducible yet but requires invalidating the entire gradle cache.

I'd therefore be in favour of just pushing people towards using `mavenLocal` and `./gradlew assembleRelease publishToMavenLocal` to test local changes